src/reader/reader.cc

```diff
@@ -247,7 +247,8 @@ void InmemReader::init_from_txt() {
     std::string bin_file = filename_ + ".bin";
     data_buf_.Serialize(bin_file);
   }
-  delete [] block_;
+  free(block_);
```

Comment:
Hi, does there any different between `delete []` and` free()`?

Reply:
I am new to C++. Please point me out when if I got anything mistaken. The difference was answered in [here](https://stackoverflow.com/questions/328834/c-delete-vs-free-and-performance). Since `_block` is a char pointer, I think the `free` operator would be a better one since a char does not have any constructor nor any destructor.